infra/docker-swarm/terraform/sg.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_security_group" "default" {
+  name        = "testground-${var.cluster}"
+  description = "Testground-${var.cluster} network security rules"
+  vpc_id      = aws_vpc.default.id
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["172.16.0.0/16"]
+  }
+
+  # outbound internet access
+  egress {
+    from_port = 0
+
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
```

Comment:
super nit: the trailing newline policy on these files is inconsistent. If TF/Ansible don't require a blank line, let's remove it from everywhere.

Reply:
Removed.